pkg/converter/convert_test.go

```diff
@@ -53,7 +53,7 @@ func TestConvert(t *testing.T) {
 			},
 		},
 		{
-			name: "convert token with msi login and override",
+			name: "convert token with msi login and override ",
```

Comment:
* nit: you have an extra whitespace in the end
* I think I was suggesting to prefix all these existing test cases with `using legacy azure auth, ...`